metastats-domain/src/main/groovy/life/qbic/metastats/request/ProjectSpecification.groovy

```diff
@@ -2,6 +2,11 @@ package life.qbic.metastats.request
 
 interface ProjectSpecification {
 
+    /**
+     * Requests the project information based on the project code
+     * @param projectCode qbic code specifying the openbis project
+     * @return
```

Comment:
I disagree. For interfaces I would make it as declarative as possible. We should fix a return type and avoid def and fill the @return documentation.

Reply:
Good Point! Is there any way to avoid repetetive(aka copy pasting the docs) javadoc documentation concerning the interface and the underlying method e.g. with @inheritDoc?

Reply:
yes, `{@inheritDoc}` should work in all implementation. This is fine as long as the behaviour is the same so especially for interfaces this fits quite well.

Reply:
I will state explicite return types/void types if the method should do the same for all interfaces

Reply:
85f417f